ParselyTrackerTests/ParselyTrackerTests.swift

```diff
@@ -20,12 +22,15 @@ class ParselyTrackerTests: ParselyTestCase {
         XCTAssertEqual(parselyTestTracker.eventQueue.length(), 0,
                        "eventQueue should be empty immediately after initialization")
         parselyTestTracker.trackPageView(url: testUrl, urlref: testUrl, metadata: nil, extraData: nil)
+        expectParselyState(self.parselyTestTracker.eventQueue.length()).toEventually(equal(1))
         XCTAssertEqual(parselyTestTracker.eventQueue.length(), 1,
                        "A call to Parsely.trackPageView should add an event to eventQueue")
```

Comment:
The XCTest assertion is now redundant.

```suggestion
        expect(self.parselyTestTracker.eventQueue.length())
            .toEventually(
                equal(1),
                description: "A call to Parsely.trackPageView should add an event to eventQueue"
            )
```